package.json

```diff
@@ -22,10 +22,10 @@
     "@babel/preset-env": "^7.6.2",
     "cross-env": "^7.0.3",
     "datocms-structured-text-to-html-string": "^1.0.13",
-    "gatsby": "^4.24.0",
-    "gatsby-plugin-image": "^2.0.0",
-    "gatsby-plugin-sharp": "^4.0.0",
-    "gatsby-transformer-remark": "^5.0.0",
+    "gatsby": "^5.0.0",
```

Comment:
@browniebroke I would change that to "^4.24.0 || ^5.0.0", what do you think?

Reply:
Is that valid syntax for a `devDependencies`?